backend/Squirrel.ConsoleApp/Squirrel.ConsoleApp.csproj

```diff
@@ -7,7 +7,20 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
-  <ItemGroup>
+	<ItemGroup>
+		<None Remove="appsettings.json" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<EmbeddedResource Include="appsettings.json">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</EmbeddedResource>
+	</ItemGroup>
+	
+	<ItemGroup>
+	<PackageReference Include="FluentValidation.AspNetCore" Version="11.1.2" />
+	<PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="6.0.21" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client.Core" Version="6.0.21" />
```

Comment:
1. Formatting
2. You first remove appsettings, then add, why?

Reply:
I added SetupProject (Installer) to my branch and copied this lines from [Max's solution](backend/Squirrel.ConsoleApp/Squirrel.ConsoleApp.csproj) to avoid conflicts